tests/test_cmt.py

```diff
@@ -1,8 +1,89 @@
 """Testing functions for the CMT services"""
 
+import strax
 import straxen
+import utilix
 import numpy as np
+from warnings import warn
+from .test_basics import test_run_id_1T
+from .test_plugins import test_run_id_nT
+from straxen.common import aux_repo
 
+def test_connect_to_db():
+    """
+    Test connection to db
+    """
+    if not straxen.utilix_is_configured():
+        warn('Cannot do test becaus  '
+             'no have access to the database.')
+        return
+
+    username=None
+    password=None
+    mongo_url=None
+    is_nt=True
+    mongo_kwargs = {'url': mongo_url,
+                        'user': username,
+                        'password': password,
+                        'database': 'corrections'}
+    corrections_collection = utilix.rundb.xent_collection(**mongo_kwargs)
+    client = corrections_collection.database.client     
+    cmt = strax.CorrectionsInterface(client, database_name='corrections')
+    df = cmt.read('global')
+    mes = 'Return empty dataframe when reading DB. Please check'
+    assert not df.empty, mes
+
+def test_1T_elife():
+    """
+    Test elife from CMT DB against historical data(aux file)
+    """
+    if not straxen.utilix_is_configured():
+        warn('Cannot do test becaus  '
+             'no have access to the database.')
+        return
+
+    elife_conf = ('elife', 'ONLINE', False)
+    elife_cmt = straxen.get_correction_from_cmt(test_run_id_1T, elife_conf)
+    elife_file = elife_conf=aux_repo + '3548132b55f81a43654dba5141366041e1daaf01/strax_files/elife.npy'
+    x = straxen.get_resource(elife_file, fmt='npy')
+    run_index = np.where(x['run_id'] == int(test_run_id_1T))[0]
+    elife = x[run_index[0]]['e_life']
+    mes = 'Elife values do not match. Please check'
+    assert elife_cmt == elife, mes
+
+def test_nT_to_pe_values(execute=True):
+    """
+    Test to_pe average values nT
+    """
+    if not straxen.utilix_is_configured() or not execute:
+        warn('Cannot do test becaus  '
+             'no have access to the database.')
+        return
+
+    gain_model = ('to_pe_model', 'ONLINE', True)
+    to_pe_cmt = straxen.get_to_pe(test_run_id_nT, gain_model, straxen.n_tpc_pmts)
+    ave_to_pe_cmt = round(np.average(to_pe_cmt), 3)
+    ave_to_pe = 0.003  # average value for the nT test run

```

Comment:
See my comment above about the test column. I could imagine that this may not be always true.

Reply:
As it says it is valid only for the nT test run. But we can get rid of this test if we get rid of `get_to_pe()`

Reply:
we remove, see below